google/cloud/spanner_v1/pool.py

```diff
@@ -171,10 +171,7 @@ def bind(self, database):
 
         while not self._sessions.full():
             resp = api.batch_create_sessions(
-                database.name,
-                self.size - self._sessions.qsize(),
-                timeout=self.default_timeout,
```

Comment:
Out of curiosity, what timeout will be used now that you've removed this parameter?

Reply:
It will use the default timeout that is specified by the spanner_client_config which is one minute:
https://github.com/googleapis/python-spanner/blob/master/google/cloud/spanner_v1/gapic/spanner_client_config.py#L44

Reply:
Ah ok thanks. If you can update the PR description, that would be great.